src/modules/accounting/accounting.py

```diff
@@ -90,14 +91,15 @@ def process_extra_data(self, blockstamp: ReferenceBlockStamp):
         self._submit_extra_data(blockstamp)
 
     def _submit_extra_data(self, blockstamp: ReferenceBlockStamp) -> None:
-        extra_data = self.lido_validator_state_service.get_extra_data(blockstamp, self.get_chain_config(blockstamp))
+        extra_data = self.get_extra_data(blockstamp)
 
-        if extra_data.extra_data:
-            tx = self.report_contract.functions.submitReportExtraDataList(extra_data.extra_data)
+        if extra_data.format == FormatList.EXTRA_DATA_FORMAT_LIST_EMPTY.value:
+            tx = self.report_contract.submit_report_extra_data_empty()
+            self.w3.transaction.check_and_send_transaction(tx, variables.ACCOUNT)
         else:
-            tx = self.report_contract.functions.submitReportExtraDataEmpty()
-
-        self.w3.transaction.check_and_send_transaction(tx, variables.ACCOUNT)
+            for tx_data in extra_data.extra_data_list:
+                tx = self.report_contract.submit_report_extra_data_list(tx_data)
+                self.w3.transaction.check_and_send_transaction(tx, variables.ACCOUNT)
```

Comment:
There will be no exception in this case. Only message in logs smth like (`Transaction reverted.`, `execution reverted: APP_AUTH_FAILED`).
So the logic is next in case of restart:
- Send all transactions one by one.
- If they locally reverted - just log it. (this part will be passed very fast)
- If transactions sent - wait for mining and wait next.

Rechecked this case manually